src/test/java/org/reaktivity/nukleus/maven/plugin/internal/ast/parse/AstParserTest.java

```diff
@@ -347,14 +347,14 @@ public void shouldParseStructWithMembers()
     @Test
     public void shouldParseStructWithListMember()
```

Comment:
Remember to rename this test to `shouldParseStructWithArrayMember`.
Also review other test names including `List`.